### PR TITLE
erase useless invoking

### PR DESCRIPTION
**Pull request by tianshapjq**
erase useless invoking in test case

### CODE REVIEW

**Comment by tianshapjq** [comment]
@piosz could u please check this pr?

File: metrics/sinks/elasticsearch/driver_test.go [hunk 1]
```diff
@@ -204,6 +204,4 @@ func TestStoreMultipleDataInput(t *testing.T) {
 		expectMsg := fmt.Sprintf(mgsTemplate, timeStr)
 		assert.Contains(t, msgsString, expectMsg)
 	}
-
-	FakeESSink = NewFakeSink()
```
Comment (andyxning) [inline metrics/sinks/elasticsearch/driver_test.go]: LGTM

Comment (andyxning) [inline metrics/sinks/elasticsearch/driver_test.go]: One nit:

you should also change `FakeESSink := NewFakeSink()` in [line 69](https://github.com/tianshapjq/heapster/blob/ac0aef2ee784f0e2b497e9208607ff61ef1f0887/metrics/sinks/elasticsearch/driver_test.go#L69) to `FakeESSink := NewFakeSink()`. 

We need all tests in `driver_test.go` to use the global `FakeESSink` variable and re-initialize it before each tests using, as we use the global `FakeESSink` in [`SaveDataIntoES_Stub`](https://github.com/tianshapjq/heapster/blob/ac0aef2ee784f0e2b497e9208607ff61ef1f0887/metrics/sinks/elasticsearch/driver_test.go#L43-L47).

**Comment by andyxning** [comment]
LGTM

**Review by AlmogBaku — APPROVED** [review]

**Comment by andyxning** [comment]
@k8s-bot ok to test

**Comment by AlmogBaku** [comment]
/assign @DirectXMan12 @piosz  

Please merge :)

**Pull request merged**